### PR TITLE
Continuous integration configured 

### PR DESCRIPTION
**Pull request by eliaset096**
Continuous Integration with Travis CLI has been configured.

### CODE REVIEW

File: cypress/fixtures/users.json [hunk 1]
```diff
@@ -229,4 +229,4 @@
       "bs": "target end-to-end models"
     }
   }
-]
+]
```
Comment (garzuzo) [inline cypress/fixtures/users.json]: add new line at the end of the file

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        ]
+    },
+    "include": [
+        "cypress/**/*.ts"
+    ],
+    "exclude": [
+        "node_modules/"
+    ]
+}
+
```
Comment (garzuzo) [inline tsconfig.json]: remove empty line

File: .nvmrc [hunk 3]
```diff
@@ -0,0 +1,2 @@
+$ node -v
+v10.16.0
```
Comment (holgiosalos) [inline .nvmrc:2]: this file just should have one line, with the version of node, it is not needed the node -v command

**Review by holgiosalos — COMMENTED** [review]
@eliaset096 pretty good job, however, for Us is more easy review one PR when it just has the content of one step. This PR has like 4 o 5 steps, sending huge PRs is not a good practices, so, for the next PR just include one section or step.

File: .nvmrc (hunk 3 above)
Comment (eliaset096) [reply .nvmrc:2]: I already corrected that.

**Comment by eliaset096** [comment]
Ok, but can I now merge this PR and upload the next step changes to a new PR?

> @eliaset096 pretty good job, however, for Us is more easy review one PR when it just has the content of one step. This PR has like 4 o 5 steps, sending huge PRs is not a good practices, so, for the next PR just include one section or step.

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**